corehq/apps/auditcare/models.py

```diff
@@ -92,7 +92,7 @@ def __str__(self):
     def create_audit(cls, request, user):
         audit = cls()
         audit.domain = get_domain(request)
-        audit.path = request.path
+        audit.path = request.path[:255]
```

Comment:
I recalled we have this: [TruncatingCharField](https://github.com/dimagi/commcare-hq/blob/2bf62b0dbb970ed0e8cea4a3c4213a9b4d557237/corehq/form_processor/models.py#L58)

Reply:
Cool, will switch to that in a follow-up PR.